app/uk/gov/hmrc/exports/connectors/ead/CustomsDeclarationsInformationConnector.scala

```diff
@@ -44,6 +44,7 @@ class CustomsDeclarationsInformationConnector @Inject()(mrnStatusParser: MrnStat
             logger.debug(s"CUSTOMS_DECLARATIONS_INFORMATION fetch MRN status response ${response.body}")
             Some(mrnStatusParser.parse(xml.XML.loadString(response.body)))
           case status =>
+            logger.debug(s"CUSTOMS_DECLARATIONS_INFORMATION fetch MRN status response ${response.body}")
```

Comment:
Is there any point logging an error at debug level?  I presume that debug isn't switched on anywhere useful

Reply:
Good point. I have changed it to `WARN`

Reply:
warn probably would be the best